src/components/common/notification-center.svelte

```diff
@@ -46,6 +58,39 @@
       <div class="title">No notifications</div>
     </div>
   {:else}
-    <p>List notifications here</p>
+    <SimplebarList classname="notification-list">
+      {#each notifications as notification (notification.id)}
+        <li class="notification">
+          <span class="content">
+            {#each getNotificationContent(notification) as fragment}
+              {#if typeof fragment === 'string'}
+                {fragment}
+              {:else if 'bold' in fragment}
```

Comment:
What if it was there but false, somehow?
```suggestion
              {:else if fragment.bold == true}
```

Reply:
We don't care, that's the beauty. This would not only impose additional useless checks, but also constrain the rules needlessly. The only thing we care about is that the `bold` attribute is there, whatever it's value may be. Besides, it will never go beyond this manual usage, do you really want to bother?

Reply:
It is just that you want the code to be easily extensible in the future. If you ever find the need to modify the fragment to include the attribute (maybe following a strict schema where all fragments are objects with specific attributes, for example), you will meet problems you don't know where they're coming from because you didn't prepare for it. And this example isn't too far fetched if we ever decided we want to use Typescript.
It is just about code design and extensibility in general.

Reply:
ok, boomer